contrib/flo-styx/src/main/java/com/spotify/flo/contrib/styx/StructuredLogMessage.java

```diff
@@ -22,32 +22,38 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 import io.norberg.automatter.AutoMatter;
-import javax.annotation.Nullable;
 
 @AutoMatter
 interface StructuredLogMessage {
-  @JsonProperty @Nullable String time();
-  @JsonProperty @Nullable String severity();
-  @JsonProperty @Nullable String logger();
-  @JsonProperty @Nullable String thread();
-  @JsonProperty @Nullable String message();
-  @JsonProperty @Nullable String framework();
-  @JsonProperty @Nullable String styx_component_id();
-  @JsonProperty @Nullable String styx_workflow_id();
-  @JsonProperty @Nullable String styx_docker_args();
-  @JsonProperty @Nullable String styx_docker_image();
-  @JsonProperty @Nullable String styx_commit_sha();
-  @JsonProperty @Nullable String styx_parameter();
-  @JsonProperty @Nullable String styx_execution_id();
-  @JsonProperty @Nullable String styx_trigger_id();
-  @JsonProperty @Nullable String styx_trigger_type();
-  @JsonProperty @Nullable String task_id();
-  @JsonProperty @Nullable String task_name();
-  @JsonProperty @Nullable String task_args();
+  @AutoMatter
+  interface Workflow {
+    @JsonProperty String styx_component_id();
```

Comment:
Would it make sense to structure things even more then?

```yaml
{
  "workflow": {
    "styx": {
      "workflow_id":"...",
      "component_id":"...",
      "workflow_id":"...",
      "docker_args":"...",
      "docker_image":"...",
      "commit_sha":"..."
    },
    "framework": "flo|luigi",
    "task": {
      "id": "...",
      "name": "...",
      "args": "..."
    }
  },
  "time": "...",
  "severity": "...",
  "logger": "...",
  "thread": "...",
  "message": "..."
}
```

Reply:
Absolutely.